### ISSUE TITLE
Download Link - Visible label and accessible name do not match

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/sul-embed#1723]
Site Improve reports the following accessibility issue with the PDF viewer:

Visible label and accessible name do not match; however, that's slightly misleading. In actuality, the label text (visual) must be included in the accessible name of the element. I suggest the following fix:

Remove the `aria-label` on the `span` element. Change the `aria-label` on the `button` element to 'Download 1 file' (or whatever the number is, but for the PDF viewer, I believe it will always be 1). Here are the before and after code snippets tested with Site Improve. 

Currently:
![Screenshot 2023-11-10 at 10 40 31 AM](https://github.com/sul-dlss/sul-embed/assets/6816624/588b944a-5549-4c9b-9a3b-063451a471a4)

Recommended Change:
![Screenshot 2023-11-10 at 10 31 41 AM](https://github.com/sul-dlss/sul-embed/assets/6816624/6a1ee3a3-a1b7-4f3c-84f6-a2e9f8feecb0)

**Comment by dbranchini** [comment]
Applies to the Geo viewer (and File viewer, but the File viewer will be redesigned).

**Comment by lwrubel** [comment]
@mjgiarlo would someone on the Media WC be interested to pick this up?

**Comment by mjgiarlo** [comment]
@lwrubel Yes, I believe so. I'll confer w/ the team and add this to our board. Thanks!

**Comment by mjgiarlo** [comment]
@aaron-collier should this issue be closed?

**Comment by aaron-collier** [comment]
@mjgiarlo waiting on the PR to be merged.
